docs/docs.polserver.com/pol100/configfiles.xml

```diff
@@ -1286,110 +1286,113 @@ Region  (string regionname)
+      [AttackHitScript       (string script for intrinsic weapon hit)]
+      [AttackMinRange        (int intrinsic weapon minimum range {default 0})]
+      [AttackMaxRange        (int intrinsic weapon maximum range {default 0})]
+      [AttackAnimation       (int animation ID {default 0 (wrestling)})]
+      [AttackProjectile      (0/1 if defined, sets the intrinsic weapon as a projectile weapon]
+      [AttackProjectileAnim  (int tile number of animation)]
+      [AttackProjectileType  (int objtype of projectile)]
+      [AttackProjectileSound (int soundID for projectile shoot)]
+      [AttackCProp           (string name, string value, will set a CProp on the intrinsicn weapon)]
+      [AttackCprop           (...)]
+
+      // Intrinsic shield creation
+      Shield                 (int intrinsic shield armor, no shield will be created if missing)
+      [ShieldOnHitScript     (string script for intrinsic shield hit)]
+      [ShieldCProp           (string name, string value, will set a CProp on the intrinsicn shield)]
+      [ShieldCprop           (...)]
+
+      [FireDamage]            (int/(die-string value))]
+      [ColdDamage]            (int/(die-string value))]
+      [EnergyDamage]          (int/(die-string value))]
+      [PoisonDamage]          (int/(die-string value))]
+      [PhysicalDamage]        (int/(die-string value))]
+
+      [MaxHp                 (int maximum hitpoints for intrinsic weapon {default 1}]
+
+      [MoveMode              (movemode string {default 'L'})]
+      [Privs                 (string privilege) [(more privs)]...]
+      [Settings              (string priv set 'on') [(more privs)]...]
+
+      [SpeechColor           (int speech color {default 0x3B2})]
+      [SpeechFont            (int speech color {default 3})]
+
+      [UseAdjustments        (0/1 {default 1})]
+      [RunSpeed              (int 0-250 {default dexterity})]
+      [SaveOnExit            (0/1 {default 1})]
+      [MethodScript          (string scriptname)]
+      [NoDropException       (0/1 {default 0})]
+
+      [LowerReagentCost       (int/(die-string value))]
+      [SpellDamageIncrease    (int/(die-string value))]
+      [FasterCasting          (int/(die-string value))]
+      [FasterCastRecovery     (int/(die-string value))]
+      [DefenceIncrease        (int/(die-string value))]
+      [DefenceIncreaseCap     (int/(die-string value))]
+      [LowerManaCost          (int/(die-string value))]
+      [HitChance              (int/(die-string value))]
+      [FireResistCap          (int/(die-string value))]
+      [ColdResistCap          (int/(die-string value))]
+      [EnergyResistCap        (int/(die-string value))]
+      [PhysicalResistCap      (int/(die-string value))]
+      [PoisonResistCap        (int/(die-string value))]
+      [Luck                   (int/(die-string value))]
+      [SwingSpeedIncrease     (int/(die-string value))]
+      [MinAttackRangeIncrease     (int/(die-string value))]
```

Comment:
Can we adjust the spacing of the existing mods to be even with the new ones so they are correctly aligned?